airgun/widgets.py

```diff
@@ -643,6 +643,8 @@ def fill(self, item):
 
 
 class Search(Widget):
+    """
+    """
```

Comment:
Please don't add empty docstrings just to make tool happy.

It's better to add this widget to list of ignored references in `docs/conf.py`:
```
nitpick_ignore = [
    ('py:class', 'airgun.widgets.Search'),
]
```

Better yet, write a sentence or two on what this widget is doing :) .